### PR TITLE
Port ReadImageData example to Python.

### PR DESCRIPTION
**Pull request by aronhelser**

### CODE REVIEW

File: src/Python/IO/ReadImageData.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    actor.SetMapper(mapper)
+    # show the edges of the image grid
+    actor.GetProperty().SetRepresentationToWireframe()
+
+    # Create the Renderer
+    renderer = vtk.vtkRenderer()
+    renderer.AddActor(actor)
+    renderer.ResetCamera()
+    renderer.SetBackground(1, 1, 1)  # Set background to white
+
```
Comment (ajpmaclean) [inline src/Python/IO/ReadImageData.py]: Use colors to set the background, e.g. 
``` python
renderer.SetBackground(colors.GetColor3d("White"))
```

**Comment by ajpmaclean** [comment]
The image file does not match the code. It has a black background not white as indicated in the code. Also consider colouring the actor.

**Comment by aronhelser** [comment]
Thanks for looking! The image does have a white background - it matches the C++ version.

I've got a weird problem - if I add a color to the actor:
```
    actor.GetProperty().SetRepresentationToWireframe()
    actor.GetProperty().SetColor(colors.GetColor3d("DarkSalmon"))
```
it works if the input .vti file has cell data, but not if it has point data, like the vase.vti file that's used for testing this example. I've also tried `SetEdgeColor` but that doesn't work either. How do I do this?

**Comment by ajpmaclean** [comment]
I was viewing the image on a white background and thought the vase was the background. So I have changed the background in the C++ example to silver. So you should change it here to Silver also.

Don't worry about adding a color to the actor. I tried a few things and the wire frame color doesn't change.

**Comment by aronhelser** [comment]
All right, should be all fixed up.

**Comment by ajpmaclean** [comment]
Thankyou

**Pull request merged**